index.ftd

```diff
@@ -1,71 +1,77 @@
--- spa.page:
-full-width: true
-sidebar: false
+-- ws.page:
 
--- spa.hero-with-image: We transform ideas into digital outcomes.
-cta-primary: Learn More
-cta-primary-link: #content
-image: $assets.files.assets.landing.svg
+-- ws.page.popup:
+
+;; to be added
```

Comment:
@priyanka9634 follow how I have added tnc popup from this repo: 
1. https://github.com/fastn-community/midnight-storm-ws/blob/ef9a968cdfe965e324bca7356fd1062f939f42ac/index.ftd#L5
2. https://github.com/fastn-community/midnight-storm-ws/blob/ef9a968cdfe965e324bca7356fd1062f939f42ac/FASTN/ws.ftd#L1
3. https://github.com/fastn-community/midnight-storm-ws/blob/ef9a968cdfe965e324bca7356fd1062f939f42ac/tnc.ftd#L1